client/src/modules/dashboard/shared/components/Layout.tsx

```diff
@@ -29,6 +31,7 @@ const links = [
     requiredPermission: Permission.GoogleAuthenticate,
   },
 ];
+const scrollWidth = 50;
```

Comment:
```suggestion
const scrollWidth = 150;
```

I didn't find issue with the functionality except with the speed of the scrolling, the scroll is slow when the `>` button is pressed.